test/e2e/specs/blocks/blocks__coblocks__extensions__gutter-control.ts

```diff
@@ -11,73 +11,90 @@ import {
+
+	let page: Page;
+	let testAccount: TestAccount;
+	let editorPage: EditorPage;
+	let pricingTableBlock: PricingTableBlock;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+		testAccount = new TestAccount( accountName );
+		editorPage = new EditorPage( page, { target: features.siteType } );
+
+		await testAccount.authenticate( page );
+	} );
+
+	it( 'Go to the new post page', async () => {
+		await editorPage.visit( 'post' );
+	} );
+
+	it( 'Insert Pricing Table block', async () => {
+		const blockHandle = await editorPage.addBlockFromSidebar(
+			PricingTableBlock.blockName,
+			PricingTableBlock.blockEditorSelector
 		);
+		pricingTableBlock = new PricingTableBlock( blockHandle );
+	} );
+
+	it( 'Open settings sidebar', async () => {
+		await editorPage.openSettings();
+	} );
+
+	skipItIf( isAtomic )( 'Verify "None" gutter is available', async () => {
+		await pricingTableBlock.setGutter( 'None' );
+	} );
+
+	it( 'Verify "Small" gutter is available', async () => {
+		await pricingTableBlock.setGutter( 'Small' );
+	} );
+
+	it( 'Verify "Medium" gutter is available', async () => {
+		await pricingTableBlock.setGutter( 'Medium' );
+	} );
+
+	it( 'Verify "Large" gutter is available', async () => {
+		await pricingTableBlock.setGutter( 'Large' );
+	} );
+
+	skipItIf( isAtomic )( 'Verify "Huge" gutter is available', async () => {
+		await pricingTableBlock.setGutter( 'Huge' );
+	} );
+
+	skipItIf( isSimple )( 'Verify "Custom" gutter is available', async () => {
+		await pricingTableBlock.setGutter( 'Custom', 2.7 );
+	} );
 
-		it( 'Set gutter to "Huge"', async () => {
-			await pricingTableBlock.setGutter( 'Huge' );
-		} );
+	it( 'Close settings sidebar', async () => {
+		await editorPage.closeSettings();
+	} );
 
-		it( 'Close settings sidebar', async () => {
-			await editorPage.closeSettings();
-		} );
+	it( 'Fill the price fields so the block is visible', async () => {
+		await pricingTableBlock.enterPrice( 1, 4.99 );
+		await pricingTableBlock.enterPrice( 2, 9.99 );
+	} );
 
-		it( 'Fill the price fields so the block is visible', async () => {
-			await pricingTableBlock.enterPrice( 1, 4.99 );
-			await pricingTableBlock.enterPrice( 2, 9.99 );
-		} );
+	it( 'Publish and visit the post', async () => {
+		await editorPage.publish( { visit: true } );
+	} );
 
-		it( 'Publish and visit the post', async () => {
-			await editorPage.publish( { visit: true } );
-		} );
+	skipItIf( isAtomic )( 'Verify the class for "Huge" gutter is present', async () => {
+		await page.locator( '.wp-block-coblocks-pricing-table .has-huge-gutter' ).waitFor();
+	} );
 
-		it( 'Verify the class for "Huge" gutter is present', async () => {
-			await page.waitForSelector( '.wp-block-coblocks-pricing-table .has-huge-gutter' );
-		} );
-	}
-);
+	skipItIf( isSimple )( 'Verify the proper value for "Custom" gutter is set', async () => {
+		await page
+			.locator( '.wp-block-coblocks-pricing-table [style="--coblocks-custom-gutter:2.7em"]' )
+			.waitFor();
```

Comment:
**Suggestion (blocking):** When I was running this locally, I was getting a strict mode violation here. It looks like there's a debug element that conflicts with this:

```
    locator.waitFor: Error: strict mode violation: locator('.wp-block-coblocks-pricing-table [style="--coblocks-custom-gutter:2.7em"]') resolved to 2 elements:
        1) <div class="wp-block-coblocks-pricing-table__inner ha…>…</div> aka locator('#post-101').getByText('4.99 9.99')
        2) <div class="wp-block-coblocks-pricing-table__inner ha…>…</div> aka locator('#debug-bar-wp-query').getByText('4.99 9.99')
```

Maybe we need to just add a `first()` here?

Reply:
This happens only when you run tests AutoProxxied. For some reason, there's this `#debug-bar-wp` being added to DOM in the mobile viewport, which is breaking the strict rule of the locator because it contains cloned target block element. I think that AutoProxxy should be disabled when running tests locally because it can change the environment to the point where some tests can break, like this one. Do you think it would be worth adding this to the general E2E readme?

Reply:
Ahhh, _that's_ what that is! Sorry for the confusion! 

> Do you think it would be worth adding this to the general E2E readme?

Yeah, that's not a bad idea!